casper/src/test/scala/coop/rchain/casper/helper/RhoSpec.scala

```diff
@@ -113,9 +117,9 @@ class RhoSpec(
         it should testName in {
           assertions.foreach {
             case RhoAssertEquals(_, expected, actual, clue) =>
-              actual should be(expected) withClue clueMsg(clue)
+               printer.buildString(actual) should be(printer.buildString(expected)) withClue clueMsg(clue)
             case RhoAssertNotEquals(_, unexpected, actual, clue) =>
-              actual should not be (unexpected) withClue clueMsg(clue)
+              printer.buildString(actual) should not be (printer.buildString(unexpected)) withClue clueMsg(clue)
```

Comment:
we should also check the underlying structure - the assertion should be twofold:
1. check pretty-printed terms match, show readable error if they don't
2. check the underlying ast matches, show the current error if they don't

2 is needed because there could be bugs in the pretty-printer and/or interpreter that would only be surfaced by comparing the AST